api/metadata.go

```diff
@@ -37,7 +37,7 @@ var metadataHandler handler = func(w http.ResponseWriter, r *http.Request) *erro
 		return errResp
 	}
 
-	metadata, err := plugin.CachedMetadata(ctx, entry)
+	metadata, err := plugin.Metadata(ctx, entry)
```

Comment:
We don't have analytics collection on Metadata?

Reply:
Nope. Should we? Seems fundamental enough that I don't know if it is necessary.

Reply:
Probably not.